signs-work-order-map/config/calcs.ts

```diff
@@ -0,0 +1,110 @@
+import { useEffect } from "react";
+import {
+  useCalculationType,
+  useSimpleCalculationType,
+  CalcComponentTypeDef,
+} from "@/types/calcs";
+
+export const MAXWIDTH = 12;
+export const MAXFEET = 1000;
+export const MAXTHICKNESS = 15;
+export const MAXRPMS = 100;
+
+export const defaultDivisors: { [key: string]: number } = {
+  thermo60: 17.5609756097561,
+  thermo90: 12.1951219512195,
+  beadsExtruded: 120,
+  primer: 3600,
+  paintGallons: 19250,
+  beadsPaint: 212.4,
+  adhesive: 3,
+};
+
+export const thermoplasticCalculations: Array<CalcComponentTypeDef> = [
+  {
+    title: "60 mils thick (maintenance)",
```

Comment:
i know these are legacy names we're porting over, but it seems like they could be cleaned up.

* **60mm thick**
* **90mm thick**
* **Beads**
* **Thermoplastic Primer**

...?

Reply:
@ChristinaTremel wondering what you think about reworking some of these titles?